### DISCUSSION TITLE
Ajouter un simulateur dirigeant SARL

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2705] [Nouveaux simulateurs]
Les gérants de SARL peuvent être :
- Majoritaire et dans ce cas sont TNS
- Minoritaire et dans ce cas sont assimilé-salarié

On peut imaginer ajouter un simulateur dirigeant SARL avec une bascule (comme IR/IS) pour afficher le bon simulateur.

**Comment by liliced** [comment]
Transformé en issue : https://github.com/betagouv/mon-entreprise/issues/3178
